Library/Formula/qrencode.rb

```diff
@@ -1,5 +1,3 @@
-require "formula"
-
```

Comment:
Similarly, probably don't want to change this as it'll show in `brew update` currently.